src/matlab/IKServer.m

```diff
@@ -26,7 +27,7 @@
     function obj = loadNominalData(obj,filename)
       if nargin < 1
         filename = [getenv('DRC_BASE'), ...
-          '/software/control/matlab/data/atlas_bdi_fp.mat'];
+          '/software/control/matlab/data/atlas_v3/atlas_bdi_fp.mat'];
```

Comment:
I'd be in favor of removing this default argument completely.  The caller should always pass in a valid path.